staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -461,7 +466,13 @@ func (h *kmsv2PluginProbe) getCurrentState() (envelopekmsv2.State, error) {
 		return envelopekmsv2.State{}, fmt.Errorf("got unexpected nil transformer")
 	}
 
-	encryptedObjectCopy := state.EncryptedObject
+	encryptedObjectCopy := kmstypes.EncryptedObject{
+		EncryptedData:          state.EncryptedObjectEncryptedData,
```

Comment:
drop the EncryptedObjectEncryptedData field (can see immediately below we never allowed it to be non-empty)